tests/unit/library/test_json_allOf.py

```diff
@@ -0,0 +1,290 @@
+from json import dumps as json_dumps
```

Comment:
What about a `tests/unit/library/json/` directory, and then this becomes `test_allOf.py` ?

Reply:
Totally agreed -- will do